src/Chunk.h

```diff
@@ -439,7 +439,12 @@ class cChunk :
 	void SetAlwaysTicked(bool a_AlwaysTicked);
 
 	// Makes a copy of the list
-	cClientHandleList GetAllClients(void) const {return m_LoadedByClient; }
+	cClientHandleList GetAllClients(void) const
+	{
+		cClientHandleList copy;
+		std::copy(m_LoadedByClient.begin(), m_LoadedByClient.end(), std::back_inserter(copy));
+		return copy;
```

Comment:
Isn't the iterator range initialization limited to iterators of the container type? That wouldn't work here, the iterators are of a different container type.
-- 
Sent from my Android phone with K-9 Mail. Please excuse my brevity.